platforms/vitalam/charts/vitalam-node/templates/service.yaml

```diff
@@ -0,0 +1,25 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: vitalam-node
+  labels:
+    app: vitalam
+spec:
+  type: ClusterIP
+  ports:
+  - name: ws
+    protocol: TCP
+    port: {{ .Values.node.ports.ws }}
+    targetPort: ws
+  - name: p2p
+    protocol: TCP
+    port: {{ .Values.node.ports.p2p }}
+    targetPort: p2p
```

Comment:
These should be on separate services.  You do not want to expose the API publicly, just internally, so it makes sense to move RPC and WS to its own service.